package.json

```diff
@@ -442,6 +449,64 @@
                     }
                 }
             }
+        ],
+        "walkthroughs": [
+            {
+                "id": "azure-get-started",
+                "title": "Get Started with Azure in VS Code",
+                "description": "Setup your account and get to know the Azure extension view.",
+                "steps": [
+                    {
+                        "id": "sign-in",
+                        "title": "Sign into your Azure account",
+                        "description": "Sign in or create an Azure Account to see all of the resources within your Azure subscription.\n[Sign In](command:azure-account.login)\n",
+                        "when": "!azure-account.signedIn",
```

Comment:
This step appears briefly and then disappears once the extension is activated. It might look nicer if we flip around the logic; i.e. the step is hidden and the context is updated to show it. However, I don't know if the focus will jump around in a weird way if this happens.

Reply:
Matt says this behavior is fine